pod/perldelta.pod

```diff
@@ -27,6 +27,15 @@ here, but most should go in the L</Performance Enhancements> section.
 
 [ List each enhancement as a =head2 entry ]
 
+=head2 Added C<newLISTOPn()>
+
+A new API function has been added that simplifies C (or XS) code that creates
+LISTOP optree fragments.  C<newLISTOPn()> is a variadic function that takes a
```

Comment:
Thinking `C<LISTOP>` could read better

Reply:
I would think so, but it seems common practice in documentation about optree-manipulating functions that the kinds of ops being worked on - BASEOP, UNOP, BINOP, etc... - don't get special formatting.

Reply:
Yes, but I think this can be corrected in the general documentation later.

Formatting helps the eyes a lot to separate those if done correctly.